src/Microsoft.AspNetCore.DataProtection.Extensions/Microsoft.AspNetCore.DataProtection.Extensions.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>Additional APIs for ASP.NET Core data protection.</Description>
-    <TargetFrameworks>net46;netstandard1.3</TargetFrameworks>
```

Comment:
.Abstractions is still 1.3. My concern is that you'll force anyone who depends on DP to move up to 2.0 even if they don't care about this new feature.